documentation/23-mariposa/README.md

```diff
@@ -0,0 +1,152 @@
+Title: Mariposa integration
```

Comment:
The RFD is using the working name "Mariposa" but it's almost certain this isn't going to be what we call the container supervisor/project service. Let's just say "Triton integration" here for now.